### DISCUSSION TITLE
Is there any discord server for support?

### DISCUSSION THREAD

**Discussion by vkamsingh** [sequelize/sequelize#15803] [Help & Questions] [answered]
hello sequelize, have any support server on discord?

**Comment by WikiRik** [comment, marked as answer]
We do not have a Discord server, and we are also not planning to get one in the future. We do have a Slack workspace; https://join.slack.com/t/sequelize/shared_invite/zt-1igt9hx8h-tGWU4AI1bRPvkRdEK5wnuQ

**Reply by ephys** [reply]
To add to this, I think it's generally better to ask questions on GitHub discussion or stack overflow, as other people with the same question can then find the answer by googling it

**Reply by Yareaj** [reply]
@WikiRik, is this still the case, or have there been any considerations for creating one?

**Reply by WikiRik** [reply]
Yes, this is still the case

**Reply by IArnaut2** [reply]
Why use millennial techbro corpo garbage instead of Discord?

**Comment by hassanshaikley** [comment]
The Slack invite link is not working!

**Reply by WikiRik** [reply]
You're right. We'll look into it!

**Reply by hassanshaikley** [reply]
Thanks!

**Reply by WikiRik** [reply]
We've updated the link and you can find it now at https://sequelize.org/slack

**Reply by Bouzomgi** [reply]
Headups: it is broken again if this group still exists.

**Reply by WikiRik** [reply]
Thanks for the message. We updated the link

**Reply by IArnaut2** [reply]
Discord is more convenient.
